users/signals.py

```diff
@@ -1,15 +1,15 @@
-from django.contrib.auth.models import User
-from django.db.models.signals import post_save
-from django.dispatch import receiver
-from .models import Profile
+# from django.contrib.auth.models import User
+# from django.db.models.signals import post_save
+# from django.dispatch import receiver
+# from .models import Profile
 
 
-@receiver(post_save, sender=User)
-def create_profile(sender, instance, created, **kwargs):
-    if created:
-        Profile.objects.create(user=instance)
+# @receiver(post_save, sender=User)
+# def create_profile(sender, instance, created, **kwargs):
+#     if created:
+#         Profile.objects.create(user=instance)
 
 
-@receiver(post_save, sender=User)
-def save_profile(sender, instance, **kwargs):
-    instance.profile.save()
+# @receiver(post_save, sender=User)
+# def save_profile(sender, instance, **kwargs):
+#     instance.profile.save()
```

Comment:
Why this code is in the comments? What is it used for?

Reply:
+1

Reply:
I can't delete it.. If I delete it I have a problem with the migration

Reply:
A comment should not affect the migration. if it does, it means something in your code, or migration process, requires a fix.

Reply:
see next comment

Reply:
What I suggest is:
- Delete the comments
- Delete the db file
- Run the migration again

I tried it with your branch locally and it looks ok

Reply:
@Davidco94 Did you try this?

Reply:
its work but I can't delete the file